src/services/fintech/router.ts

```
@@ -6,40 +6,69 @@
-    const id = this.currencyProviders[currency] ?? "flutterwave";
-    const provider = this.providers.get(id);
-    if (provider) return provider;
-    const fallback = this.providers.get("flutterwave");
-    if (fallback) return fallback;
+  /**
+   * Selects the best provider for a currency based on health and fees.
+   * If all fail, throws an error.
+   * @param currency
+   * @param opts Optionally pass {simulateOutageFor?: FintechProviderId}
+   */
+  async getProvider(currency: string, opts?: { simulateOutageFor?: FintechProviderId }): Promise<FintechProvider> {
```

ESLint

[error] Replace `currency:·string,·opts?:·{·simulateOutageFor?:·FintechProviderId·}` with `⏎····currency:·string,⏎····opts?:·{·simulateOutageFor?:·FintechProviderId·},⏎··`

(prettier/prettier)